config/routes.rb

```diff
@@ -214,6 +214,7 @@
   get 'u/:username/'                       => 'users#show',       as: :user
 
   get 'dashboard'                          => 'application#boot_angular_ui', as: :dashboard
+  get 'dashboard/show_muted'               => 'application#boot_angular_ui', as: :muted
```

Comment:
Alternatively, you could add a `filter_dashboard_path` helper with `as: :filter_dashboard`, and call it if we did end up needing it for something reason
```
filter_dashboard_path(:show_muted)
```